doc/api/webstreams.md

```diff
@@ -1497,5 +1666,49 @@ added: v16.7.0
 * Returns: {Promise} Fulfills with the contents of the stream parsed as a
   UTF-8 encoded string.
 
+```mjs
+import { json, text, blob, buffer } from 'node:stream/consumers';
+import { Readable } from 'node:stream';
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
+// Consumes data stream as the text: '[{"message": "hello world from consumers!"},{...},{...}]'
+const data = await text(readable);
+console.log(`from readable: ${data.length}`); // From readable: 42000001
+```
+
+```cjs
+const { text } = require('node:stream/consumers');
+const { Readable } = require('node:stream');
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+async function readFromStream() {
+  const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
+  // Consumes data stream as the text: '[{"message": "hello world from consumers!"},{...},{...}]'
+  const data = await text(readable);
```

Comment:
I'd consume data from a text string instead of a Stringified JSON so people could understand the difference between `json` and `text` functions